pages/projects/life-journeys.js

```diff
@@ -0,0 +1,140 @@
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+import { CallToAction } from "../../components/molecules/CallToAction";
+
+function HTMList({ tag = "ul", content, listClassName, liClassName }) {
+  const parseList = (content) =>
+    content
+      .split("*") // split the string on asterisks
+      .filter((item) => item) // filter out empty strings
+      .map((item, index) => (
+        <li className={liClassName} key={index}>
+          {item.trim()}
+        </li>
+      ));
+
+  return tag === "ul" ? (
+    <ul className={listClassName}>{parseList(content)}</ul>
+  ) : (
+    <ol className={listClassName}>{parseList(content)}</ol>
+  );
+}
+
+export default function LifeJourneys(props) {
+  const { t } = useTranslation("common", "dc");
+  const { asPath } = useRouter();
+
+  return (
+    <Layout
+      locale={props.locale}
+      langUrl={asPath}
+      breadcrumbItems={[
+        { text: "Service Canada Labs", link: "/" },
+        { text: "Projects", link: "/projects" },
+      ]}
+    >
+      <Head>
+        {process.env.NEXT_PUBLIC_ADOBE_ANALYTICS_URL ? (
+          <script src={process.env.NEXT_PUBLIC_ADOBE_ANALYTICS_URL} />
+        ) : (
+          ""
+        )}
+        <title>{t("lj:scLabsLifeJourneys")}</title>
+        <link rel="icon" href="/favicon.ico" />
+        <meta name="dcterms.title" content={t("lj:scLabsLifeJourneys")} />
+        <meta
+          name="dcterms.language"
+          content={props.locale === "en" ? "eng" : "fra"}
+        />
+        <meta name="dcterms.creator" content={t("creator")} />
+        <meta name="dcterms.accessRights" content="2" />
+        <meta name="dcterms.service" content="ESDC-EDSC_SCLabs-LaboratoireSC" />
+      </Head>
+      <section className="layout-container relative mb-10">
+        <h1
+          id="pageMainTitle"
+          className="mb-8 text-h1l font-bold flex-wrap"
+          tabIndex="-1"
+        >
+          {t("lj:lifeJourneysTitle")}
+        </h1>
+        <h2 className="mb-6 font-bold leading-10">
+          {t("lj:lifeJourneysHeading1")}
+        </h2>
+        <p className="mb-6">{t("lj:lifeJourneysContent1")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent2")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent3")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent4")}</p>
+        <h2 className="mb-6 font-bold leading-10">
+          {t("lj:lifeJourneysHeading2")}
+        </h2>
+        <figure className="mb-6 border shadow-experiment-shadow">
+          <img
+            src={
+              props.locale === "en"
+                ? "/having-a-childEN.png"
+                : "/having-a-childFR.png"
+            }
+            alt={t("lj:lifeJourneysImgAltText1")}
+          ></img>
+          <figcaption className="p-2.5 text-sm font-display border-t">
+            {t("lj:lifeJourneysImgCaption1")}
```

Comment:
This caption seems to be missing in French.

Reply:
Thanks, was missing the `Img` part of the string in the French JSON.